docs/ui/auth/fragments/vue/authenticator.md

```diff
@@ -57,6 +57,53 @@ Amplify UI Components use `slots` based off of the [Web Components slot element]
 | `totp-setup`           | Content placed inside of the totp-setup workflow for when a user opts to use TOTP MFA                                  |
 | `greetings`            | Content placed inside of the greetings navigation for when a user is signed in                                         |
 
+### Custom Form Fields
+
+If you'd like to customize the form fields in the Authenticator Sign In or Sign Up component you can do so by using the `formFields` property.
+
+The following is an example highlights the use of Authenticator with customized Sign Up form fields and [authentication with email](#authenticate-with-email-or-phone-number):
+
+```html
+<amplify-authenticator username-alias="email">
+  <amplify-sign-up
+    slot="sign-up"
+    username-alias="email"
+    :form-fields.prop="[
```

Comment:
I've never looked closely at Vue code. This looks horrendous haha. Why is the object in quotes?! Does Vue deserialize object props, which are passed as strings?! ... is there a compile step? What if users want to provide references, instead of serializable data?

Reply:
We should actually move this to the component `data` member.